src/screens/ProfileScreen/elements/KarmaScore.tsx

```diff
@@ -1,35 +1,101 @@
-        {score}
-      </Text>
-      <Text style={{fontSize: 12, fontWeight: '700', color: colors.white}}>Karma</Text>
-    </View>
+    <TouchableWithoutFeedback onPress={() => setIsTooltipShown(true)}>
+      <View
+        style={{
+          borderRadius: 32,
+          flexDirection: 'row',
+          justifyContent: 'center',
+          alignItems: 'center',
+          alignSelf: 'flex-start'
+        }}>
+        <View style={{marginRight: 8}}>
+          <CircleGradient size={normalize(28)} width={normalize(2.3)} fill={score}>
+            <FastImage
+              resizeMode={FastImage.resizeMode.center}
+              source={BetterScore}
+              style={{width: 28, height: 28}}
+            />
+          </CircleGradient>
+        </View>
+        <Text
+          style={{
+            fontWeight: '500',
+            fontSize: 24,
+            lineHeight: 36,
+            marginRight: 8
+          }}>{`${score} Karma`}</Text>
+        <Tooltip
+          allowChildInteraction={true}
+          isVisible={isTooltipShown}
+          placement={'bottom'}
+          backgroundColor={'rgba(0,0,0,0)'}
+          closeOnContentInteraction={false}
+          onClose={() => setIsTooltipShown(false)}
+          closeOnBackgroundInteraction={true}
+          contentStyle={{
+            borderRadius: 10,
+            padding: 16,
+            height: 'auto'
+          }}
+          content={
+            <View>
+              <Text
+                style={{
+                  fontWeight: '600',
+                  fontSize: normalize(16),
+                  lineHeight: normalize(24),
+                  marginBottom: normalize(8)
+                }}>
+                Your Karma Score
+              </Text>
+              <Text
+                style={{
+                  fontWeight: '400',
+                  fontSize: normalize(12),
+                  lineHeight: normalize(18),
+                  marginBottom: normalize(8),
+                  color: '#69707C'
+                }}>
+                The higher your Karma score, the higher your visibility on the platform. Get
+                rewarded for positive contributions to the community - and avoid being blocked by
+                others!
+              </Text>
+              <Text
+                style={{
+                  fontWeight: '600',
+                  paddingVertical: normalize(8),
+                  fontSize: normalize(14),
+                  lineHeight: normalize(20),
+                  color: '#4782D7',
+                  width: '100%',
+                  textAlign: 'right'
+                }}
+                onPress={() => setIsTooltipShown(false)}>
+                Got it!
+              </Text>
+            </View>
+          }>
+          <View style={{}}>
+            <MemoIc_question_mark width={normalize(13)} height={normalize(13)} />
+          </View>
+        </Tooltip>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Tooltip` component has been introduced to provide additional information about the karma score. This is a good UX practice. However, make sure that the tooltip's content and behavior are consistent with the overall design and functionality of the application.

```diff
-  <Text style={{fontWeight: '800', color: colors.white, fontSize: 21, marginRight: 8}}>
-    {score}
-  </Text>
+  <Text
+    style={{
+      fontWeight: '500',
+      fontSize: 24,
+      lineHeight: 36,
+      marginRight: 8
+    }}>{`${score} Karma`}</Text>
```
The text styling and content have been updated. Ensure that these changes align with the overall design system of the application and that the new font size, weight, and line height values are accessible and readable across different screen sizes and resolutions.


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
ignore